es6-shim.js

```diff
@@ -1438,7 +1438,7 @@
       if (number === 0) {
         return 32;
       }
-      return 32 - (number).toString(2).length;
+      return 32 - 1 - Math.floor(Math.log(number) * Math.LOG2E + (1 / 4294967296));
```

Comment:
`Math.floor(Math.log(value) * Math.LOG2E)` is computed with some `error` and we need to fix this error. If we add something bigger than error, than `floor` should give values not lesser, than true result. But this something should not too big, otherwise we will get value one bigger, than true result. 
Also, it seems to me, `log(integer + 1) - log(integer) >= log(MAX_INTEGER+ 1) - log(MAX_INTEGER) >= 1 / (MAX_INTEGER + 1)`.
So "something" = `1 / (MAX_INTEGER + 1) / 2` should be good.
Seems, `Math.log` should be very inaccurate (with relative error > 2000 ulp (?)) to make this implementation incorrect.